burgerpy/modules/qpick.py

```diff
@@ -0,0 +1,39 @@
+from hashlib import sha1
+import urllib2
+import simplejson as json
+from burgerpy.common import Module, Config
+
+# picks order according to quantum RNG --- entangles user with a photon in Australia, basically.
+class QuantumPickModule(Module):
+    def on_qpick(self, channel, method, props, body):
+            data = json.loads(body)
+            words = data["content"].split()
+            origin = data["channel"]
+
+            if not words:
+                return
+
+            result = self.pick(words)
+            msg = self.compose_msg(origin, "%s: %s" % (data["from"], result))
+            return self.send_result(data["source"], msg)
+
+    def pick(self, words):
+        numbers=[]
+        while len(numbers)<len(words): # repeats are possible
+            try:
+                retrieved=json.loads(urllib2.urlopen("https://qrng.anu.edu.au/API/jsonI.php?length="+int(len(words)-len(numbers))+"&type=uint16").read())
```

Comment:
Maybe read one value and use it as seed for PRNG? There can be an awful lot of queries.